nitrite.go

```diff
@@ -459,28 +482,3 @@ func checkECDSASignature(
 
 	return ecdsa.Verify(publicKey, hashSigStruct, r, s)
 }
-
-func NewDocumentFromCosePayloadBytes(bytes []byte) (*Document, error) {
```

Comment:
Became `Timestamp` after moving a newer version of this function from our PR to `hf/nitrite`